ansible_tower_client.gemspec

```diff
@@ -19,10 +19,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  # HACK: Rails 5 dropped support for Ruby < 2.2.2
+  active_support_version = "< 5" if Gem::Version.new(RUBY_VERSION) <= Gem::Version.new("2.2.2")
+  spec.add_runtime_dependency "activesupport", active_support_version
```

Comment:
My only problem with this - It would be great if we could pull out active_support all together - granted if we rolled our own version of `camelize` - which may not be trivial.

Outside of that - I would like to pull out the term #HACK: in the comment.  The change will probably live there for a long time - and #HACK: seems to indicate this change is extremely temporary.